app/models/content_store_payload_version.rb

```diff
@@ -0,0 +1,41 @@
+class ContentStorePayloadVersion < ActiveRecord::Base
+  def self.increment(content_item_id:)
```

Comment:
That's true enough in your tests, but in the real world usage you're passing `content_item.id` so reading the full signature in the caller, it's currently something like `ContentStorePayloadVersion.increment(content_item_id: content_item.id)` which feels odd when you could just `ContentStorePayloadVersion.increment(content_item.id)`. So the provided argument forms part of the signature.